tools/services

```diff
@@ -17,58 +17,76 @@
 # You should have received a copy of the GNU General Public License
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
-function storage_service()
+function detect_deployment()
 {
-    {
-      sudo $ACTION archivematica-storage-service
-    } || { # catch, try with uwsgi in case this is 1.5 style deployment
-        {
-            sudo service uwsgi $ACTION
-        } || {
-            echo "storage_service $ACTION did not work"
-        }
-    }
+    # detect if upstart/systemd/sysvinit
+    if [[ `/sbin/init --version 2> /dev/null` =~ upstart ]]; then
+	INITSYS="upstart";
+    elif [[ `systemctl` =~ -\.mount ]]; then
+	INITSYS="systemd";
+    elif [[ `-f /etc/init.d/cron && ~ -h /etc/init.d/cron` ]]; then
+	INITSYS="sysvinit" # unlikely
+    fi
+
+  # detect if 1.5 style deployment
```

Comment:
To be consistent with the other script comments, I think you should remove the initial blank space after `#`